scripts/vpn_connect.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+INTERVAL=10
+RETRY=6
+
+/usr/local/pulse/PulseClient_x86_64.sh -h $VPN_HOST -u $VPN_USER -p $VPN_PASS -r Users &
```

Comment:
This command may leak credentials: 

```
Checking for missing dependency packages for command line client ...
executing command : /usr/local/pulse/pulsesvc -h 183.84.2.165 -u xxxx -p yyyyy -r Users
```
Should redirect the output into > /dev/null

Reply:
done